libbeat/docs/processors-config.asciidoc

```diff
@@ -233,6 +233,7 @@ The supported actions are:
  * <<drop-fields,`drop_fields`>>
  * <<drop-event,`drop_event`>>
  * <<add-cloud-metadata,`add_cloud_metadata`>>
+ * <<json-decode-fields,`json_decode_fields`>>
```

Comment:
The name of the processors is `decode_json_fields` instead of `json_decode_fields`.